workloads/network-perf-v2/README.md

```
@@ -14,47 +14,69 @@ Running from CLI:
 This will orchestrate multiple network performance tests for OpenShift ServiceMesh.
 
-| Test | Parallelism | MessageSize | Iterations | Duration | 
-|------|------------|------------|----------|
-|TCP Stream| 1 - 2 - 4 | 64 - 1024 - 8192 | 2 | 30 |
-|TCP RR| 1 | 64 - 1024 - 8192 | 2 | 30 |
+| Test | Parallelism | MessageSize | Samples | Duration |
+|------|-------------|-------------|---------|----------|
+| TCP Stream | 1, 2, 4 | 64, 1024, 8192 | 2 | 30 |
+| TCP RR | 1 | 64, 1024, 8192 | 2 | 30 |
+
+### MicroShift
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)